### ISSUE TITLE
Support for Noxon iRadio

### ISSUE THREAD

**Issue by x789** [x789/xonox#18]
My iRadio is running Firmware-Version 3.8.36.6923, Bootloader-Version 6834 (available here http://ultron.info/NOXON/NOXON_iRadio/Update/Beta/).

The iRadio DNS queries _radio567.vtuner.com_, _radio5672.vtuner.com_, _www.terratec.com_ and _www.my-noxon.com_ - there are NO DNS queries to _noxonserver.eu_.  These domains are being redirected (via PiHole).

Using another tool I was able to monitor the iRadio web requests.  Initially I suspected the issue was related to the domain name or a configuration issue, but after further investigation it appears the iRadio is requesting a different URL that the one in your code.

Here are the 3 online queries and the URL's

```
Internet Radio = /setupapp/radio567/asp/BrowseXPA/LoginXML.asp
Podcasts = /setupapp/radio567/asp/BrowseXPA/loginXML.asp
My Noxon = /RadioNativeLogin.php
```

For reference the image linked below is the iRadio model being used.

https://www.noxon.ch/wp-content/uploads/2017/10/noxon-iradio-300x215.jpg

Finally: I've run strings on the 4 most recent firmware versions from _http://ultron.info/NOXON/NOXON_iRadio/Update/_ and the two firmware versions available at _https://www.noxon.ch/support/downloads/_ (under iRadio) - they all contain the previous listed 4 domains and none contain any _noxonserver.eu_ domains.

_Originally posted by @Hack5190 in https://github.com/x789/xonox/issues/13#issuecomment-1916276959_

**Comment by x789** [comment]
The ReadMe claims that _xonox_ is compatible with the _Noxon iRadio_ (without number). Based on your post, I don't think that's true. Please apologize for the confusion.
The _iRadio_ uses different endpoints than the _iRadio 300_ and the _Noxon 2_ (`BrowseXPA` instead of `BrowseXML`).
Since I do not own an _iRadio_ device and therefore cannot run any tests, I would need your help to add support for the _iRadio_ to _xonox_. Would you consider support?

**Comment by x789** [comment]
I released the version 1.1.0 of _xonox_ that adds support for Noxon 2 Audio and Noxon iRadio.
@Hack5190: Does it work when you redirect radio567.vtuner.com and radio5672.vtuner.com  to your _xonox_-instance?
